### ISSUE TITLE
Duplicate deadchat messages after using !spectate wolfchat on

### ISSUE THREAD

**Issue by fndax** [lykoss/lykos#485]
After using `!spectate wolfchat on` while joined to deadchat, I started getting messages twice, in two formats:

```
01:44:01 -- lykos: misha says: hm...
01:44:02 -- lykos: [deadchat] misha says: hm...
01:44:19 -- lykos: Nay says: lol
01:44:19 -- lykos: [deadchat] Nay says: lol
```

Doing `!spectate wolfchat off` didn't fix the duplicates.

**Comment by fndax** [comment]
Nay triggered the same issue, then left deadchat with !leave. They continued getting messages prefixed with `[deadchat]`, but stopped getting the ones without.

**Comment by JeDaYoshi** [comment]
Logs for the last case:
```
21:44:14 <Nay> spectate off
21:44:15 <lykos> [deadchat] Nay says: spectate off
21:44:15 <lykos> Usage: spectate <wolfchat> [on|off]
21:44:19 <Nay> lol
21:44:19 <lykos> [deadchat] Nay says: lol
21:44:19 <Nay> leave
21:44:20 <lykos> [deadchat] Nay says: leave
21:44:20 <lykos> You have left the deadchat.
21:44:20 <lykos> [deadchat] Nay has left the deadchat.
21:44:24 <lykos> [deadchat] dax says: misha, wanna report this one too or shall i?
21:44:26 <lykos> [deadchat] misha says: hm yeah only real admins can spec deadchat i think
21:44:28 <Nay> join
21:44:28 <lykos> [deadchat] Nay has joined the deadchat.
21:44:29 <lykos> You are now in the deadchat.
21:44:29 <lykos> Players: misha, dax, and Nay
21:44:30 <lykos> minetester has joined the deadchat.
21:44:30 <lykos> The game has ended, check #werewolf for the results.
```

**Comment by skizzerz** [comment]
Fixed in 52bfcc63bde6dae3530c94f46bfb7ed7e53f6bb2
